io.github.opennumismat.OpenNumismat.yml

```diff
@@ -0,0 +1,126 @@
+app-id: io.github.opennumismat.OpenNumismat
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: open-numismat
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=home
+  - --filesystem=/media      # for export to mobile
+  - --filesystem=/run/media  # for export to mobile
+  - --device=dri
+#  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --env=TMPDIR=/var/tmp
+
+build-options:
+  cflags: -O2 -g
+  cxxflags: -O2 -g
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+
+modules:
+  - python3-requirements.json
+
+  # https://github.com/flathub/io.qt.qtwebengine.BaseApp/tree/branch/6.5/krb5
+  - name: krb5
+    subdir: src
+    cleanup:
+      - /bin
+      - /share/et
+      - /share/examples
+      - /share/man
+    config-opts:
+      - --localstatedir=/var/lib
+      - --sbindir=${FLATPAK_DEST}/bin
+      - --disable-rpath
+      - --disable-static
+    post-install:
+      - install -Dm644 ../krb5.conf -t ${FLATPAK_DEST}/etc/
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/flathub/io.qt.qtwebengine.BaseApp/branch/6.5/krb5/krb5.conf
+        sha256: 031b3ddfd6db31b61bd086298374ba43422a4b1bbd84176ad252796711814c2a
+      - type: archive
+        url: https://kerberos.org/dist/krb5/1.20/krb5-1.20.1.tar.gz
+        sha256: 704aed49b19eb5a7178b34b2873620ec299db08752d6a8574f95d41879ab8851
+
+  - name: pypi-dependencies
+    buildsystem: simple
+    build-commands: []
+    config-opts:
+      - --force-clean
+    modules:
+      - name: python3-PySide6
```

Comment:
so where is Qt?

Reply:
What do you mean? PySide not require additional Qt installation

Reply:
see https://github.com/flathub/flathub/pull/5829#issuecomment-2466211364